app/info/[infoID]/page.tsx

```
@@ -0,0 +1,27 @@
+import { DateFormat } from "@/components/DateFormat";
+import { MultiLine } from "@/components/MultiLine";
+import { getInfo } from "@/db/getInfo";
+import { notFound } from "next/navigation";
```

ESLint

[error] Run autofix to sort these imports!

(simple-import-sort/imports)